docs/_user_guide/best_practices_monitor_usage.md

```diff
@@ -152,127 +127,59 @@ system.time({
-
-![](/images/monitor-2.png)
-
-Last modification we are going to make is to pass the number of cores as a command line argument to our R script.
-Save the following to a new script called `investment-npv-parallel-args.R`.
-
-```R
-#!/usr/bin/env Rscript
-############################################
-# This script accepts a user specified argument to set the number of cores to run on
-# Run from the command line:
-#
-#      Rscript investment-npv-parallel-args.R 8
-#
-# this will execute on 8 cores
-###########################################
-# accept command line arguments and save them in a list called args
-args = commandArgs(trailingOnly=TRUE)
-library(foreach)
-library(doParallel)
+!!! Notes
+    If you have not installed `foreach` and `doParallel` packages. You have to run these commands after loading `r`: `install.packages("foreach")` and `install.packages("doParallel")`
 
-options(warn=-1)
-# set the number of cores here from the command line. Avoid using detectCores() function.
-ncore <- as.integer(args[1])
-
-# register parallel backend to limit threads to the value specified in ncore variable
-registerDoParallel(ncore)
+Once the program is running, monitor your usage with `userload` command in the second window:
 
-# define function for NPV calculation
-npv_calculation <- function(cashflows, discount_rate) {
-  # inputs: cashflows (a vector of cash flows over time) and discount_rate (the discount rate).
-  npv <- sum(cashflows / (1 + discount_rate)^(0:length(cashflows)))
-  return(npv)
-}
+```bash title="Terminal Command"
+watch -n 1 userload
+```
 
-# number of trials
-num_trials <- 500000
+While the program is running, you should see about 8 CPU cores being utilized in `userload` output. Note that the `watch` command allows you to see the real time CPU cores count every 1 second.
 
-# measure the execution time of the Monte Carlo simulation
-system.time({
-  # use the foreach package to loop through the specified number of trials (num_trials) in parallel
-  # within each parallel task, random values for input parameters (cash flows and discount rate) are generated for each trial
-  # these random input values represent different possible scenarios
-  results <- foreach(i = 1:num_trials, .combine = rbind) %dopar% {
-    # randomly generate input values for each trial
-    cashflows <- runif(10000, min = -100, max = 100)  # random cash flow vector over 10,000 time periods.
-    # these cash flows can represent costs (e.g., initial investment) and benefits (e.g., revenue or savings) associated with the project
-    discount_rate <- runif(1, min = 0.05, max = 0.15)  # random discount rate at which future cash flows are discounted
+![Resource monitoring during parallel run](/assets/images/monitor_userload_r_8_cores.png)
 
-    # calculate NPV for the trial
-    npv <- npv_calculation(cashflows, discount_rate)
+Run `htop -u $USER` in the third window, where `$USER` is your SUNetID:
```

Comment:
Your example shows the process tree, which is not shown by default.  Maybe add a little note about changing to that mode in htop/